moveit_core/CMakeLists.txt

```diff
@@ -184,44 +183,34 @@ add_subdirectory(kinematics_metrics)
 add_subdirectory(dynamics_solver)
 add_subdirectory(collision_detection_bullet)
 
-pluginlib_export_plugin_description_file(moveit_core collision_detector_bullet_description.xml)
-
-# TODO: Port python bindings
-# add_subdirectory(python)
-# set(pymoveit_libs
-#   moveit_collision_detection
-#   moveit_kinematic_constraints
-#   moveit_planning_scene
-#   moveit_python_tools
-#   moveit_robot_model
-#   moveit_robot_state
-#   moveit_transforms
-# )
+pluginlib_export_plugin_description_file(
+  moveit_core collision_detector_bullet_description.xml)
+
+# TODO: Port python bindings add_subdirectory(python) set(pymoveit_libs
+# moveit_collision_detection moveit_kinematic_constraints moveit_planning_scene
+# moveit_python_tools moveit_robot_model moveit_robot_state moveit_transforms )
 #
-# pybind_add_module(pymoveit_core
-#     python/pymoveit_core.cpp
-#     collision_detection/src/pycollision_detection.cpp
-#     robot_model/src/pyrobot_model.cpp
-#     robot_state/src/pyrobot_state.cpp
-#     transforms/src/pytransforms.cpp
-#     planning_scene/src/pyplanning_scene.cpp
-#     kinematic_constraints/src/pykinematic_constraint.cpp
-# )
-# target_include_directories(pymoveit_core SYSTEM PRIVATE ${catkin_INCLUDE_DIRS})
-# target_link_libraries(pymoveit_core PRIVATE ${pymoveit_libs} ${catkin_LIBRARIES})
+# pybind_add_module(pymoveit_core python/pymoveit_core.cpp
+# collision_detection/src/pycollision_detection.cpp
+# robot_model/src/pyrobot_model.cpp robot_state/src/pyrobot_state.cpp
+# transforms/src/pytransforms.cpp planning_scene/src/pyplanning_scene.cpp
+# kinematic_constraints/src/pykinematic_constraint.cpp )
+# target_include_directories(pymoveit_core SYSTEM PRIVATE
+# ${catkin_INCLUDE_DIRS}) target_link_libraries(pymoveit_core PRIVATE
+# ${pymoveit_libs} ${catkin_LIBRARIES})
 #
-# #catkin_lint: ignore_once undefined_target (pymoveit_core is defined by pybind_add_module)
-# install(TARGETS pymoveit_core LIBRARY DESTINATION ${CATKIN_GLOBAL_PYTHON_DESTINATION})
-
+# #catkin_lint: ignore_once undefined_target (pymoveit_core is defined by
+# pybind_add_module) install(TARGETS pymoveit_core LIBRARY DESTINATION
+# ${CATKIN_GLOBAL_PYTHON_DESTINATION})
 
 install(
   TARGETS ${THIS_PACKAGE_LIBRARIES}
   EXPORT export_${PROJECT_NAME}
   LIBRARY DESTINATION lib
   ARCHIVE DESTINATION lib
   RUNTIME DESTINATION bin
-  INCLUDES DESTINATION include
-)
+  INCLUDES
+  DESTINATION include)
```

Comment:
Is that a bug in the formatter or do the few characters make the line too long? :confused: